service/src/main/java/com/commercetools/pspadapter/payone/PaymentHandler.java

```diff
@@ -73,7 +73,8 @@ public PaymentHandleResult handlePayment(@Nonnull final String paymentId) {
         } catch (final Exception e) {
             return handleException(e, paymentId);
```

Comment:
Does it make sense to log the retrycounter here too

Reply:
@ahalberkamp yeah, totally. My previous git push has missed some changelogs. Please check now :)